<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sidang</class>
 <widget class="QMainWindow" name="Sidang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sidang</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableData">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>270</y>
      <width>661</width>
      <height>221</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Judul</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Jadwal</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mahasiswa</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dosen</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="inptID">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="inptJudul">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Judul Skripsi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Jadwal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mahasiswa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dosen</string>
    </property>
   </widget>
   <widget class="QComboBox" name="inptMhs">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>130</y>
      <width>165</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>200</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdate">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>200</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>523</x>
      <y>200</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>200</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="inptJadwal">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>60</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::DaySection</enum>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
   </widget>
   <widget class="QComboBox" name="inptDosen">
    <property name="geometry">
     <rect>
      <x>258</x>
      <y>130</y>
      <width>165</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>inptDosen</zorder>
   <zorder>inptJadwal</zorder>
   <zorder>tableData</zorder>
   <zorder>inptID</zorder>
   <zorder>inptJudul</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>inptMhs</zorder>
   <zorder>btnDelete</zorder>
   <zorder>btnUpdate</zorder>
   <zorder>btnSave</zorder>
   <zorder>btnReset</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
